woocommerce/woocommerce-gutenberg-products-block
# Deactivate nightly builds

## What

Fixes https://github.com/woocommerce/woocommerce/issues/43070

## Why



Currently, we're still generating nightly builds, each night at 12:01 AM, despite that the WooCommerce Blocks repo has been merged into the Woo core monorepo. This is resource-inefficient and creates unnecessary noise in https://a8c.slack.com/archives/C02UBB1EPEF. 

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

> [!IMPORTANT]
> **Due to the natuer of this change, we should merge this PR and then wait for 24h to see the change. No new nightly builds should occur via https://github.com/woocommerce/woocommerce-blocks/releases/tag/nightly.**


## Screenshots or screencast



n/a

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


n/a
